esb-connector-creatio-crm/src/test/resources/artifacts/ESB/config/proxies/creatio/createContact.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Copyright (c) 2022, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+  ~
+  ~ WSO2 Inc. licenses this file to you under the Apache License,
+  ~ Version 2.0 (the "License"); you may not use this file except
+  ~ in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~ http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing,
+  ~ software distributed under the License is distributed on an
+  ~ "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+  ~ KIND, either express or implied.  See the License for the
+  ~ specific language governing permissions and limitations
+  ~ under the License.
+  ~
+  -->
+<proxy xmlns="http://ws.apache.org/ns/synapse"
+       name="postContact"
+       startOnLoad="true"
+       statistics="disable"
+       trace="disable"
+       transports="http,https">
+<target>
+    <inSequence>
+        <property name="httpMethod" value="post"/>
+        <creatiocrm.init>
+            <hostName>https://hostname.creatio.com</hostName>
+            <password>password</password>
+            <username>username</username>
+            <timeout>2000</timeout>
+        </creatiocrm.init>
+        <creatiocrm.contactCreate/>
+        <respond/>
+    </inSequence>
+    <outSequence/>
+    <faultSequence/>
+</target>
+<description/>
+</proxy>
```

Comment:
shall we add EOD. ( extra line at the end ) so that this red icon goes away. Usually we do that for all the files.

Reply:
Added new line